tasks/src/conversational/data.ts

```diff
@@ -0,0 +1,75 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A dataset of 7k conversations explicitly designed to exhibit multiple conversation modes: displaying personality, having empathy, and demonstrating knowledge.",
+			id:          "blended_skill_talk",
+		},
+		{
+			description: "ConvAI is a dataset of human-to-bot conversations labeled for quality. This data can be used to train a metric for evaluating dialogue systems",
+			id:          "conv_ai_2",
+		},
+		{
+			description: "EmpatheticDialogues, is a dataset of 25k conversations grounded in emotional situations",
+			id:          "empathetic_dialogues",
+		},
+	],
+	demo: {
```

Comment:
I got your page up on my local, here's what schema looks like:
<img width="730" alt="Ekran Resmi 2022-10-07 13 15 30" src="https://user-images.githubusercontent.com/53175384/194540809-2a324944-a8f2-4bff-8357-7c9fbeb30362.png">

The conversational pipeline takes a dialogue as input and returns next round of conversation, so output should be a single text. 
Example: 

```
from transformers import pipeline, Conversation
pipe = pipeline("conversational")

conversation_1 = Conversation("Going to the movies tonight - any suggestions?")
conversation_2 = Conversation("What's the last book you have read?")
pipe([conversation_1, conversation_2])

```
(you can commit this code snippet as example snippet if you wish 🙂)
Output looks like below:

```
[Conversation id: 8f2f3cce-858a-4bdf-82dc-8972f3f458da 
 user >> Going to the movies tonight - any suggestions? 
 bot >> The Big Lebowski ,
 Conversation id: 30c10891-c55f-4a89-a22d-bb18253af0fc 
 user >> What's the last book you have read? 
 bot >> The Last Question ]
```
So as output you can write a simple sentence instead.

Reply:
You need build of website itself, and the website is closed source so you can't get a local build. I got the build and everything seemed fine 🙂